pkg/scheduler/cache/fake_eventhandler.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2021 The Kubernetes Authors.
```

Comment:
replace 'Kubernetes' to `Volcano`

Reply:
There are a large number of files in the stock code file using `kubernetes` in License, even [missing](https://github.com/volcano-sh/volcano/blob/master/pkg/scheduler/actions/elect/elect.go). Whether to consider providing script to refresh and verification?